magento/pwa-studio
# [chore?]: Use createCustomer gql mutation instead of create-account REST endpoint

## Description

This change replaces the old, unused email validation function *and* the `createAccount` redux/rest action with usage of the `createCustomer` mutation.

## Related Issue





Closes #1210.

## Acceptance 




### Verification Steps


1. Go to `/create-account` and try to create an account with an email that is already used. You should see the new error.
2. Go to the create account view in left nav and do the same. You should see the error.
3. Now, in both views, try to create accounts with valid emails and it should succeed.

## Screenshots / Screen Captures (if appropriate)
![](https://i.gyazo.com/f8e17ecd6b8af5725cd25906e1113d11.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
